### ISSUE TITLE
Tests error on aarch64, ppc64le, s390x 

### ISSUE THREAD

**Issue by eclipseo** [disintegration/gift#20]
Golang 1.12.6 on aarch64, ppc64le, s390x:
```
Testing    in: /builddir/build/BUILD/gift-1.2.0/_build/src
         PATH: /builddir/build/BUILD/gift-1.2.0/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/gift-1.2.0/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/disintegration/gift/version=1.2.0 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/disintegration/gift
github.com/disintegration/gift
--- FAIL: TestBrightness (0.00s)
    colors_test.go:434: test [brightness (-30)] failed: image.Rectangle{Min:image.Point{X:0, Y:0}, Max:image.Point{X:5, Y:3}}, []byte{0x0, 0x0, 0x0, 0x0, 0x0, 0x14, 0x64, 0x54, 0x64, 0x14, 0x0, 0x34, 0x0, 0x34, 0x0}
--- FAIL: TestGolden (0.61s)
    gift_test.go:593: resulting image differs from golden: contrast_increase
    gift_test.go:593: resulting image differs from golden: saturation_decrease
    gift_test.go:593: resulting image differs from golden: saturation_increase
    gift_test.go:593: resulting image differs from golden: hue_rotate
FAIL
```

**Comment by disintegration** [comment]
Thank you for the report! It should be fixed now.

The Go compiler now implements a fused multiply and add (FMA) instruction on some architectures, which leads to floating-point rounding differences compared to amd64.

https://golang.org/ref/spec#Floating_point_operators

I've modified the golden tests by adding a tolerance when comparing two images on architectures other than amd64. As for for brightness test, I've adjusted the input to avoid rounding differences.
